app/scss/_navbar.scss

```diff
@@ -0,0 +1,36 @@
+.navbar {
+  display: flex;
+  flex: 1;
+  justify-content: flex-end;
+
+  &__menu {
+    display: none;
+  }
+
+  li {
+    margin-right: 1em;
```

Comment:
I suggest to use same trick with ":not(:last-child)" here too. 
You will save 1 line of code

Reply:
it can be in two ways
In both ways, I delete what's in the media query
```
li {
  &:not(:last-child) {
    margin-right: 1em;
  }

  &:last-child {
    margin-left: auto;
  }
}
```
or
```
li {
  margin: 0 1em 0 0;

  &:last-child {
    margin: 0 0 0 auto;
  }
}
  ```
What can I use?

Reply:
From the functionality point of view, you can use any of them. They both do the same thing. But in current project I suggest to use the first option, because in this case you do not overwrite all margins for the last child, but just set up the required margins for appropriate elements.